Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    class FixMultiplication
+    {
+        private static float value1, value2, answer;
+        private static string temp1 = null, temp2 = null;
```

Comment:
Why do you need static variables at class level..?
Also please follow the naming conventions.
Static instances remain alive throughout the life of application..
Your purpose is to find the digit , upon that it should dispose all the variables and instances used by your method..
You can read more about static and garbage disposal in c#